app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -46,6 +46,13 @@ class AccountNotFoundError < StandardError; end
 
     private
 
+    def merge_patient_info(form526_submission)
```

Comment:
```suggestion
    def patient_info(form526_submission)
```

Reply:
I agree but what is the reasoning here, to avoid verbs?